### PR TITLE
Add `g TAB` as a substitute on terminal for Tab.

### PR DESCRIPTION
**Pull request by crocket**
`g TAB` feels intuitive.

### CODE REVIEW

**Comment by jojojames** [comment]
Looks reasonable. @Ambrevar ?

File: readme.org [hunk 1]
```diff
@@ -189,9 +189,12 @@ more.
...
          - Tab key is recognized as ~<tab>~ in GUI and ~TAB~ in terminal.
            ~TAB~ equals ~C-i~.
-         - Since ~C-i~ is bound to jumping forward for vim compatibility,
-           bind ~g[~ and ~g]~ to the functions that Shift+Tab and Tab
-           are bound to on GUI for terminal compatibility.
+         - ~C-i~ is bound to jumping forward for vim compatibility.
+           If Shift+Tab is not relevant, just bind ~g TAB~ to the function
+           that Tab is bound to. If Shift+Tab is relevant,
+           bind ~g]~ and ~g TAB~ to the function that Tab is bound to,
+           and bind ~g[~ or ~g}~ to the function that Shift+Tab is bound to
```
Comment (Ambrevar) [inline readme.org]: Why `g}`?

File: readme.org [hunk 2]
```diff
@@ -189,9 +189,12 @@ more.
        - Tab key
          - Tab key is recognized as ~<tab>~ in GUI and ~TAB~ in terminal.
            ~TAB~ equals ~C-i~.
-         - Since ~C-i~ is bound to jumping forward for vim compatibility,
-           bind ~g[~ and ~g]~ to the functions that Shift+Tab and Tab
-           are bound to on GUI for terminal compatibility.
+         - ~C-i~ is bound to jumping forward for vim compatibility.
```
Comment (Ambrevar) [inline readme.org]: vim -> Vim.

**Comment by Ambrevar** [comment]
Looks good to me beside the comments in the README.  Thanks!

File: readme.org (hunk 1 above)
Comment (crocket) [reply readme.org]: Because Shift+Tab may not be functionally related to Tab in some modes such as org mode.

**Comment by Ambrevar** [comment]
But why not just `g[` then?

**Comment by crocket** [comment]
Because `g[` feels like something that does something opposite to what `g]` does.

**Comment by Ambrevar** [comment]
Isn't Shift-Tab the opposite of Tab?

**Comment by crocket** [comment]
Not always. In org-mode, Shift-Tab is not functionally opposite to Tab.

**Comment by Ambrevar** [comment]
So what are you suggesting?  That we use `g}` in Org mode for `shift-Tab`?

`g[` would be OK to me here too.  I find it simpler to have just one key instead of two.

**Comment by crocket** [comment]
Ok. I'm going to omit `g}` since `g}` is hard to press anyway.

**Comment by jojojames** [comment]
@crocket Sorry if I missed this, is this ready?

**Comment by crocket** [comment]
I think it is ready.

**Comment by jojojames** [comment]
Sorry @crocket , merging.

**Pull request merged**